dataikuapi/dssclient.py

```diff
@@ -629,6 +629,48 @@ def create_global_api_key(self, label=None, description=None, admin=False):
         key = resp.get('key', '')
         return DSSGlobalApiKey(self, key)
 
+    ########################################################
+    # Personal API Keys
+    ########################################################
+
+    def list_personal_api_keys(self):
+        """
+        List all personal API keys:
+            - not admin: only the keys belonging to the owner of the key
+            - admin: All the personal keys
+
+        :returns: All personal API keys, as a list of dicts
+        """
+        return self._perform_json(
+            "GET", "/personal-api-keys/")
+
+    def get_personal_api_key(self, key):
+        """
+        Get a specific API key from the user doing the request
+
+        :param str key: the secret key of the desired API key
+        :returns: A :class:`dataikuapi.dss.admin.DSSPersonalApiKey` API key handle
+        """
+        return DSSPersonalApiKey(self, key)
+
+    def create_personal_api_key(self, label=""):
+        """
+        Create a Personal API key corresponding to the user doing the request, and return a handle to interact with it
+
+        :param: label string (optional): Label for the new API key
+        :returns: A :class:`dataikuapi.dss.admin.DSSPersonalApiKey` API key handle
+        """
+        resp = self._perform_json(
+            "POST", "/personal-api-keys/", body={"label": label})
+        if resp is None:
+            raise Exception('API key creation returned no data')
+        if resp.get('messages', {}).get('error', False):
+            raise Exception('API key creation failed : %s' % (json.dumps(resp.get('messages', {}).get('messages', {}))))
+        if not resp.get('id', False):
+            raise Exception('API key creation returned no key')
+        key = resp.get('key', '')
+        return DSSPersonalApiKey(self, key)
```

Comment:
We will force people using the result of this method to change their code. Before it was returning a Dict, where they could do `my_created_key['key']`  (same for the other fields).

Now we need the `get_definition()` to get the same dict as before.

We might inherit a dict to avoid the need for our customers to migrate their code.